modules/simple-payments/paypal-express-checkout.js

```diff
@@ -100,8 +120,11 @@ var PaypalExpressCheckout = {
 		if ( ! paypal ) {
 			throw new Error( 'PayPal module is required by PaypalExpressCheckout' );
 		}
+		if ( ! jQuery ) {
```

Comment:
Hmm, this should normally not happen. We require `jQuery` as a dependency of this script. Also, not sure we should error the whole page if jQuery is not defined.